contracts/StrategyVersionRegistry.vy

```diff
@@ -0,0 +1,90 @@
+# @version 0.2.12
+governance: public(address)
+pendingGovernance: public(address)
+
+event NewGovernance:
+    governance: address
+
+event NewPendingGovernance:
+    governance: address
+
+event StrategyRegistered:
+    strategy: address
```

Comment:
would it make sense to add this as indexed arg?

Reply:
cc @salazarguille